beacon-chain/p2p/fork_watcher.go

```diff
@@ -23,6 +25,12 @@ func (s *Service) forkWatcher() {
 				if err != nil {
 					log.WithError(err).Error("Could not add fork entry")
 				}
+
+				// from Bellatrix Epoch, the MaxGossipSize and the MaxChunkSize is changed to 10MB.
+				if currEpoch == params.BeaconConfig().BellatrixForkEpoch {
```

Comment:
While it is fine to change this via the forkwatcher, how does this work when prysm starts up after the bellatrix fork epoch ? Prysm will revert to the old gossip max size and chunk size values and will be incompatible with the networking spec again.